api/reference/1.0/extensions/create.md

```diff
@@ -4,38 +4,26 @@ title: Create
 
 # Create an Extension
 
-Add an Extension to a Property. Installation is an
+Add an Extension to a Property. Installation is **not** an
```

Comment:
I think we should maybe just remove the whole paragraph.  Even the description here is kinda teeing it up to explain what used to happen when this was async.

Reply:
Thanks for the review; changes have been done.